encoding/masking-limiting-etc.md

```diff
@@ -22,14 +22,57 @@ This article will cover:
 
 Masking refers to a broad set of techniques used to merge multiple clips,
 usually one filtered clip merged with a source clip
-according to an overlay mask.
+according to an overlay mask. 
 A mask clip may contain any information
 generated from a pixel-wise operation.
+Mask clips are usually grayscale, 
+i.e. they consist of only one plane and thus contain no color information.
+In VapourSynth, such clips use the color family `GRAY`
+and one of these formats: 
+`GRAY8` (8 bits integer),
+`GRAY16` (16 bits integer),
+or `GRAYS` (single precision floating point).
+
+
 Vapoursynth includes some basic tools for manipulating masks as well:
 
 [**std.Minimum/std.Maximum**](http://www.vapoursynth.com/doc/functions/minimum_maximum.html)
 
-TODO
+The Minimum/Maximum operations replace each pixel
+with the smallest/biggest value in its 3x3 neighbourhood. 
+The 3x3 neighbourhood of a pixel
+are the 8 pixels directly adjacent to the pixel in question
+plus the pixel itself.
+To illustrate:
+![3x3 neighborhood](images/3x3.png)
```

Comment:
This would probably be better as a figure surrounded by blank lines and the caption "Illustration of the 3x3 neighborhood" without referring to it with a colon.